.github/workflows/cpp-ttnn-project.yaml

```diff
@@ -57,11 +57,11 @@ jobs:
             -w ${{ github.workspace }}
           run: |
             set -eu # basic shell hygiene
-            ./build_metal.sh --disable-unity-builds --build-type Release
+            ./build_metal.sh --build-type Release
 
       # TTNN project
       - name: Checkout cpp-ttnn-project-template
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
was this an initial error with using v3?

Reply:
No.  The error was fixed [here](https://github.com/tenstorrent/tt-metal/pull/14504/files).

And since @tt-rkim 's comment [here](https://github.com/tenstorrent/tt-metal/pull/14442#discussion_r1824986776) came in post-merge, I just bring it along for a ride in this PR.